content/en/docs/setup/best-practices/certificates.md

```diff
@@ -57,6 +59,9 @@ Required CAs:
 | front-proxy-ca.crt,key | kubernetes-front-proxy-ca | For the [front-end proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/) |
 
 On top of the above CAs, it is also necessary to get a public/private key pair for service account management, `sa.key` and `sa.pub`.
+The diagram below illustrates the CA key and certificate files shown in the previous table that you need to provide to use a single root CA from which all other keys and certificates are generated:
+
+![Certificates for single root CA](/images/docs/Certs-SingleRootCA.svg)
```

Comment:
I like the idea of adding a visual representation of the required certificates, but if I can express a personal opinion about the selected format:
- the /etcd nested folder in some cases is not easy to spot (especially in the graphs with many certificates) 
- in some cases it is not easy not spot cert/key pairs (some of them are one above he other, some one of the left to the others)
- I would suggest to remove the legend/the color convention (the type of each file is already embedded in the file extension)

Also, the diagram is reporting the sa key/pub, which are not included in the table above, please remove

Reply:
What about a simpler representation, like the one you achieve with tree?
```
|-- pki
|   |-- ca.crt
|   |-- ca.key
|   |-- ...
|   |-- etcd
|   |   |-- ca.crt
|   |   |-- ca.key
```

This is has the advantage that it is already know by most of the users, and also the nice side effect that it cab be achieved without adding images to the web site...

Reply:
I appreciate your candid opinion. I've had other suggestions to simplify how the files are represented. I think you are right. I'll make that change to the diagrams.